frontend/public/components/storage-class-form.tsx

```diff
@@ -55,7 +57,111 @@ export class StorageClassForm_ extends React.Component<
+        pool: {
+          name: 'Pool',
+          hintText: 'Ceph pool into which the RBD image shall be created',
+          required: true,
+        },
+        imageFormat: {
+          name: 'Image Format',
+          hintText: 'Ceph RBD image format',
+          values: { '2': '2' },
+          required: true,
+        },
+        imageFeatures: {
+          name: 'Image Features',
+          hintText: 'Ceph RBD image features',
+          values: { layering: 'layering' },
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-name': {
+          name: 'Provisioner Secret Name',
+          hintText: 'The name of provisioner secret',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-namespace': {
+          name: 'Provisioner Secret Namespace',
+          hintText: 'The namespace where provisioner secret is created',
+          required: true,
+        },
+        'csi.storage.k8s.io/node-stage-secret-name': {
+          name: 'Node Stage Secret Name',
+          hintText: 'The name of Node Stage secret',
+          required: true,
+        },
+        'csi.storage.k8s.io/node-stage-secret-namespace': {
+          name: 'Node Stage Secret Namespace',
+          hintText: 'The namespace where Ceph is deployed',
+          required: true,
+        },
+        'csi.storage.k8s.io/fstype': {
+          name: 'Filesystem Type',
+          hintText: 'Ceph RBD filesystem type. Default set to ext4',
+          required: true,
+        },
+      },
+    },
+    'cephfs.csi.ceph.com': {
+      title: 'Ceph FS',
+      provisioner: 'cephfs.csi.ceph.com',
+      documentationLink: 'https://rook.github.io/docs/rook/master/ceph-filesystem.html',
+      parameters: {
+        clusterID: {
+          name: 'Cluster ID',
+          hintText: 'The namespace where Ceph is deployed',
+          required: true,
+        },
+        pool: {
+          name: 'Pool',
+          hintText: 'Ceph pool into which the RBD image shall be created',
+        },
+        fsName: {
+          name: 'Filesystem Name',
+          hintText: 'CephFS filesystem name into which the volume shall be created',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-name': {
+          name: 'Provisioner Secret Name',
+          hintText: 'The name of provisioner secret',
+          required: true,
+        },
+        'csi.storage.k8s.io/provisioner-secret-namespace': {
+          name: 'Provisioner Secret Namespace',
+          hintText: 'The namespace where provisioner secret is created',
+          required: true,
+        },
+        'csi.storage.k8s.io/node-stage-secret-name': {
+          name: 'Node Stage Secret Name',
+          hintText: 'The name of Node Stage secret',
+          required: true,
+        },
+        'csi.storage.k8s.io/node-stage-secret-namespace': {
+          name: 'Node Stage Secret Namespace',
+          hintText: 'The namespace where Ceph is deployed',
```

Comment:
this need to be updated as https://github.com/openshift/console/pull/3172/files#diff-2d573e2f986c3bea9d6365a7a40ee904R147